meetup-organizing.md

```diff
@@ -0,0 +1,38 @@
+# Organizing a meetup
+
+The following details the basic process for working with the Quorum to plan a meetup, confirm a speaker and topic, and provide any needed information or materials for the meetup event.
+
```

Comment:
**Suggestion (non-blocking):** Maybe add a quick opening sentence that says something like: "Thank you for participating in Quorum and for being willing to host an upcoming event!"